src/ripple/app/paths/RippleState.h

```diff
@@ -24,8 +24,10 @@
 #include <ripple/protocol/Rate.h>
 #include <ripple/protocol/STAmount.h>
 #include <ripple/protocol/STLedgerEntry.h>
+
 #include <cstdint>
-#include <memory>  // <memory>
+#include <memory>
```

Comment:
Is this _still_ needed now that we eliminated the `std::make_shared`?

Reply:
fixed in `875f1e28c8 [fold] Rm unused include file`